src/common/points.rs

```diff
@@ -133,6 +136,60 @@ impl Validate for UpdateOperation {
     }
 }
 
+impl StrictModeVerification for UpdateOperation {
+    fn query_limit(&self) -> Option<usize> {
+        None
+    }
+
+    fn timeout(&self) -> Option<usize> {
+        None
+    }
+
+    fn indexed_filter_read(&self) -> Option<&segment::types::Filter> {
+        None
+    }
+
+    fn indexed_filter_write(&self) -> Option<&segment::types::Filter> {
+        None
+    }
+
+    fn request_exact(&self) -> Option<bool> {
+        None
+    }
+
+    fn request_search_params(&self) -> Option<&segment::types::SearchParams> {
+        None
+    }
+
+    fn check_strict_mode(
+        &self,
+        collection: &Collection,
+        strict_mode_config: &StrictModeConfig,
+    ) -> Result<(), CollectionError> {
+        match self {
+            UpdateOperation::Delete(delete_op) => delete_op
+                .delete
+                .check_strict_mode(collection, strict_mode_config),
```

Comment:
This one may delete by filter (others may operate by filter too).

Do we need to check it above, or does this nesting take care of it checking it in the `DeleteOperation` itself?

Reply:
What do you mean by checking here?
`StrictModeVerification` is implemented for `PointsSelector` with a check for writing filter.